CASMPET-7619/CASMPET-7620/CASMPET-7621/CASMPET-7630/CASMPET-7632: Create/fix procedures for changing iSCSI workers (#6132)
Cray-HPE/docs-csm: install/configure_administrative_access.md

This procedure sets up resources in Kubernetes (a Kubernetes Secret and ConfigMap) which are later
applied to the management nodes using CFS node personalization in section

> [7. Configure management nodes with CFS](#9-configure-management-nodes-with-cfs) below.

Reviewer: Link text needs to match the destination
```suggestion
[9. Configure management nodes with CFS](#9-configure-management-nodes-with-cfs) below.
```

This procedure writes the `root` password hash and SSH keys to Vault which are later
applied to the management nodes using CFS node personalization in section

> [7. Configure management nodes with CFS](#9-configure-management-nodes-with-cfs) below.

Reviewer: Link text needs to match the destination
```suggestion
[9. Configure management nodes with CFS](#9-configure-management-nodes-with-cfs) below.
```

SUCCESS
```

## 8. iSCSI SBPS configuration

> In CSM 1.6, all the worker nodes were configured and enabled as iSCSI SBPS targets. In CSM 1.7.0, selective
> node personalization is introduced where if user/admin wants to limit some worker nodes to be configured as
> iSCSI targets, then it requires to create an HSM group named `iscsi_worker` and add the worker node xnames
> which are intended to be configured as iSCSI targets to this group. This has to be done during `Configure
> administrative access`.

Reviewer: ```suggestion

node personalization is supported. All worker nodes are still **configured** for iSCSI, but selective node personalization gives administrators control over which worker nodes are actually enabled as iSCSI SBPS targets.
The default behavior is still the same as in CSM 1.6, so if no action is taken to use this feature,
then all worker nodes will be enabled as iSCSI targets.
```

> administrative access`.

Reviewer: ```suggestion
node personalization is supported. All worker nodes are still **configured** for iSCSI, but selective node
personalization gives administrators control over which worker nodes are actually enabled as iSCSI SBPS
targets. The default behavior is still the same as in CSM 1.6, so if no action is taken to use this feature,
then all worker nodes will be enabled as iSCSI targets.
```

> For details, see [Managing selective node personalization](../operations/iscsi_sbps/Managing_selective_node_personalization.md) and [worker node personalization](../operations/iscsi_sbps/iscsi_sbps.md#1-worker-node-personalization)

Reviewer: ```suggestion
For administrators who do not wish to use this feature, no action is required, and this step can be skipped.
Otherwise, before proceeding with the install, follow the procedure in the
[CSM install](../operations/iscsi_sbps/Managing_selective_node_personalization.md#csm-install) section
of [Managing Selective Node Personalization](../operations/iscsi_sbps/Managing_selective_node_personalization.md).
```

> 

Reviewer: ```suggestion

Later in the install process,
[SAT Bootprep](../operations/system_admin_toolkit/usage/SAT_Bootprep.md)
adds the iSCSI configuration layer to the NCN CFS configurations. If the above procedure to enable this feature has
not been done when that CFS configuration is applied to the NCNs, then all of the worker nodes will be enabled as
iSCSI targets.

```

SUCCESS
```

## 8. iSCSI SBPS configuration

In CSM 1.6, all the worker nodes were configured and enabled as iSCSI SBPS targets. In CSM 1.7.0, selective
node personalization is supported. All worker nodes are still **configured** for iSCSI, but selective node
personalization gives administrators control over which worker nodes are actually enabled as iSCSI SBPS targets.
The default behavior is still the same as in CSM 1.6, so if no action is taken to use this feature, then all
worker nodes will be enabled as iSCSI targets.

For administrators who do not wish to use this feature, no action is required, and this step can be skipped.

> Otherwise, before proceeding with the install, follow the procedure in the [CSM install](../operations/iscsi_sbps/Managing_selective_node_personalization.md#csm-install) section of [Managing Selective Node Personalization](../operations/iscsi_sbps/Managing_selective_node_personalization.md).

Reviewer: Break up long line.
```suggestion
Otherwise, before proceeding with the install, follow the procedure in the
[CSM install](../operations/iscsi_sbps/Managing_selective_node_personalization.md#csm-install) section of
[Managing Selective Node Personalization](../operations/iscsi_sbps/Managing_selective_node_personalization.md).
```

SUCCESS
```

## 8. iSCSI SBPS configuration

> In CSM 1.6, all the worker nodes were configured and enabled as iSCSI SBPS targets. In CSM 1.7.0, selective

Reviewer: ```suggestion
In CSM 1.6, all the worker nodes were configured and enabled as iSCSI SBPS targets. Starting in CSM 1.7.0, selective
```
